kordac/KordacExtension.py

```diff
@@ -10,32 +9,48 @@
 from kordac.processors.RemoveTitlePreprocessor import RemoveTitlePreprocessor
 from kordac.processors.SaveTitlePreprocessor import SaveTitlePreprocessor
 from kordac.processors.GlossaryLinkPattern import GlossaryLinkPattern
-from kordac.processors.ButtonLinkBlockProcessor import ButtonLinkBlockProcessor
-from kordac.processors.BoxedTextBlockProcessor import BoxedTextBlockProcessor
 from kordac.processors.BeautifyPostprocessor import BeautifyPostprocessor
 from kordac.processors.ConditionalProcessor import ConditionalProcessor
 from kordac.processors.RemovePostprocessor import RemovePostprocessor
 from kordac.processors.JinjaPostprocessor import JinjaPostprocessor
 from kordac.processors.HeadingBlockProcessor import HeadingBlockProcessor
-from kordac.processors.FrameBlockProcessor import FrameBlockProcessor
-from kordac.processors.TableOfContentsBlockProcessor import TableOfContentsBlockProcessor
 from kordac.processors.ScratchTreeprocessor import ScratchTreeprocessor
 from kordac.processors.ScratchCompatibilityPreprocessor import ScratchCompatibilityPreprocessor
+from kordac.processors.GenericTagBlockProcessor import GenericTagBlockProcessor
+from kordac.processors.GenericContainerBlockProcessor import GenericContainerBlockProcessor
 
 from kordac.utils.UniqueSlugify import UniqueSlugify
 from kordac.utils.HeadingNode import HeadingNode
 from kordac.utils.overrides import is_block_level, BLOCK_LEVEL_ELEMENTS
 
-from collections import defaultdict
+from collections import defaultdict, OrderedDict
 from os import listdir
 import os.path
 import re
 import json
 
 from jinja2 import Environment, PackageLoader, select_autoescape
 
+
 class KordacExtension(Extension):
+    '''The Kordac markdown extension which enables all the processors,
+    and extracts all the important information to expose externally to
+    the Kordac converter.
+    '''
+
     def __init__(self, processors=[], html_templates={}, extensions=[], *args, **kwargs):
```

Comment:
a processor further down defined args and kwargs, should they be defined in this doc string too? or taken out of the other?